python/pyspark/sql/functions.py

```diff
@@ -2325,6 +2325,41 @@ def check_string_field(field, fieldName):
     return Column(res)
 
 
+def session_window(timeColumn, gapDuration):
+    """
+    Generates session window given a timestamp specifying column.
+    Session window is the one of dynamic windows, which means the length of window is vary
```

Comment:
is vary -> is varying?

Reply:
Session window is the one of dynamic windows -> Session window is one of dynamic windows